src/audio/module_adapter/module_adapter.c

```diff
@@ -865,8 +864,7 @@ static int module_adapter_audio_stream_type_copy(struct comp_dev *dev)
 	struct comp_buffer __sparse_cache *sinks_c[PLATFORM_MAX_STREAMS];
 	struct processing_module *mod = comp_get_drvdata(dev);
 	struct list_item *blist;
-	uint32_t num_input_buffers = 0;
-	uint32_t num_output_buffers = 0;
+	uint32_t num_input_buffers, num_output_buffers;
 	int ret, i = 0;
 
 	if (mod->stream_copy_single_to_single)
```

Comment:
there is a similar code in pipe 2.0 DP PR2, how about move i = 0; to before line above line 873? this will make line 873 - 880 and 882 - 889 looks symmetry。